src/app/models/annotated-platewellids.ts

```diff
@@ -0,0 +1,3 @@
+export interface AnnotatedPlatewellids {
+  result: Array<{'jnjs': string, 'id': string, 'smiles': string, 'zhang': number}>;
```

Comment:
not sure what you mean here - under /models there is actually a representation of the **response object** from each API endpoint, that's why we refer to its properties/fields and define their type accordingly.
`result` is the only property of the outer object, so if it gets removed in the definition of the model we shouldn't be able to parse the object and get the needed values, inside the components.